src/components/Book.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+function Book({ title, author }) {
+  return (
+    <div>
+      <h2>
+        Title :
+        {' '}
+        {title}
+      </h2>
+      <h4>
+        Author :
+        {' '}
+        {author}
+        {' '}
+      </h4>
+    </div>
+  );
```

Comment:
- [ ] The book should contain a remove button, kindly implement this so the user can remove the book from the list (the button just needs to be rendered, no need to be functional at the moment)